### ISSUE TITLE
[com_postinstall] Invalid markup due to truncation

### ISSUE THREAD

**Issue by Quy** [joomla/joomla-cms#15095]
### Steps to reproduce the issue
Go to Components > Post-install Messages.
View page source.
Scroll to the bottom of the page.
Locate the following invalid markup `<a</p>` due to truncation.
```
<p class="lead">As part of our post-release review process for the 3.6.4 release, the Joomla! Security Strike Team has identified and confirmed an additional side effect of the issue resolved in <a</p>...						</div>
```

**Comment by RonakParmar** [comment]
Confirming the issue.
Joomla! 3.7.0-rc1
![screen shot 2017-04-05 at 04 17 07](https://issues.joomla.org/uploads/1/0a174def474eb7be39c2c22d393fff12.png)<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15095">issues.joomla.org/tracker/joomla-cms/15095</a>.</sub>

**Comment by franz-wohlkoenig** [comment]
closed having PR #15170 <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15095">issues.joomla.org/tracker/joomla-cms/15095</a>.</sub>
